docs/installation/configuration/cmake_options.rst

```diff
@@ -318,6 +311,10 @@ The building and execution of these tests is specified by the *Fast DDS* CMake o
           using the *Fast DDS* DDS-layer API.
         - ``ON`` ``OFF``
         - ``OFF``
+    *   - :class:`FASTDDS_EXAMPLE_TESTS`
+        - Enables the building of example tests for the verification of the Fast DDS examples |br|
```

Comment:
```suggestion
        - Enables the building of example tests for the validation of the Fast DDS examples |br|
```

Reply:
We decide to leave it as the same formulation is used in more places